hs_core/templates/pages/my-resources.html

```diff
@@ -353,92 +353,95 @@ <h4 class="modal-title" id="myModalLabel">Create New Label</h4>
+                                            <input type="hidden" name="label_type" value="SAVEDLABEL">
+                                            <input type="hidden" name="action" value="DELETE">
+                                        </form>
+                                        <span data-label="{{ label }}" data-form-type="delete-label"
+                                              class="btn-label-remove glyphicon glyphicon-remove"
+                                              data-form-id="form-delete-label-{{ label }}"></span>
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                        {% else %}
+                            <tr class="no-items-found">
+                                <td>No labels found.</td>
+                            </tr>
+                        {% endif %}
+                        </tbody>
+                    </table>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                </div>
             </div>
         </div>
     </div>
-</div>
 
-<!-- Delete Resource Modal -->
-<div class="modal fade" id="delete-multiple-resources-dialog" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
-     aria-hidden="true">
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                <h4 class="modal-title" id="delete-resource-title">Delete Resources</h4>
-            </div>
-            <div class="modal-body">
-                <div>Select resources to delete</div>
-            </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-                <button id="btn-delete-multiple-resources" class="btn btn-danger" data-dismiss="modal">Confirm</button>
+    <!-- Delete Resource Modal -->
+    <div class="modal fade" id="delete-multiple-resources-dialog" tabindex="-1" role="dialog"
+         aria-labelledby="myModalLabel"
+         aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title" id="delete-resource-title">Delete Resources</h4>
+                </div>
+                <div class="modal-body">
+                    <div>Select resources to delete</div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
```

Comment:
ID attribute. We should have id attribute for every html tag.

Reply:
I disagree. I think id attributes should only be added if there is a need to specifically reference the element somewhere else.

Reply:
I agree with @Maurier - only write code that we use.  If functionality that comes along that needs the ids, this can be added as needed.

Reply:
selenium automation need the id for referencing UI element to emulate user interaction. I observed Neal doing this. Without ID, the automation code is hard to write and difficult to maintain. Maybe there is another way that help to automatically generate random id for each element to facilitate automation testing. I would discuss this with @ndebuhr separately.